src/generateTokenKey.sh

```diff
@@ -1,4 +1,4 @@
-key=`head -c 20 /dev/random | base64`
-echo "${key}"
-sed -i "s@<token-key>@$key@g" UI/.env.local;
-sed -i "s@<token-key>@$key@g" API/.envrc;
+key=`head -c 20 /dev/random | base64`
+echo "${key}"
+sed -i "s@<token-key>@$key@g" UI/.env.local;
+sed -i "s@<token-key>@$key@g" API/.envrc;
```

Comment:
Fixing line endings?

Reply:
Yes, putting them all back to linux line endings so they run correctly.